lib/components/Pivot/Pivot.tsx

```diff
@@ -1,11 +1,9 @@
 import * as React from 'react';
 import * as classNames from 'classnames/bind';
```

Comment:
nit: import classNames from 'classnames/bind';

Reply:
This is how it is imported everywhere, if we do this change we should do it all over the place